docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md

```diff
@@ -19,18 +19,19 @@ This page contains a list of available feature toggles. To learn how to turn on
-| `cloudWatchDynamicLabels`    | Use dynamic labels instead of alias patterns in CloudWatch datasource                                           | Yes                |
-| `prometheusBufferedClient`   | Enable buffered (old) client for Prometheus datasource as default instead of streaming JSON parser client (new) |                    |
-| `internationalization`       | Enables internationalization                                                                                    | Yes                |
-| `accessTokenExpirationCheck` | Enable OAuth access_token expiration check and token refresh using the refresh_token                            |                    |
+| Feature toggle name                 | Description                                                                                                     | Enabled by default |
+| ----------------------------------- | --------------------------------------------------------------------------------------------------------------- | ------------------ |
+| `promQueryBuilder`                  | Show Prometheus query builder                                                                                   | Yes                |
+| `disableEnvelopeEncryption`         | Disable envelope encryption (emergency only)                                                                    |                    |
+| `database_metrics`                  | Add Prometheus metrics for database tables                                                                      |                    |
+| `lokiMonacoEditor`                  | Access to Monaco query editor for Loki                                                                          | Yes                |
+| `featureHighlights`                 | Highlight Grafana Enterprise features                                                                           |                    |
+| `commandPalette`                    | Enable command palette                                                                                          | Yes                |
+| `cloudWatchDynamicLabels`           | Use dynamic labels instead of alias patterns in CloudWatch datasource                                           | Yes                |
+| `prometheusBufferedClient`          | Enable buffered (old) client for Prometheus datasource as default instead of streaming JSON parser client (new) |                    |
+| `internationalization`              | Enables internationalization                                                                                    | Yes                |
+| `accessTokenExpirationCheck`        | Enable OAuth access_token expiration check and token refresh using the refresh_token                            |                    |
+| `disablePrometheusExemplarSampling` | Disable the sampling of Prometheus's exemplar sampling                                                          |                    |
```

Comment:
Can this be: "Disable Prometheus examplar sampling" and remove the first 'sampling'? Not sure if that change alters the technical meaning, so wanted to ask.